### PR TITLE
DTSSTCI-36

### PR DESCRIPTION
**Pull request by tomxelliott**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DTSSTCI-36

### Change description ###
Create Case Welsh translations

### CODE REVIEW

File: src/main/resources/check-your-answers/translation.json [hunk 1]
```diff
@@ -30,37 +30,45 @@
...
-        "representativeOrganisationName": "Organisation or business name - welsh ",
-        "representativeContactNumber": "Contact number - welsh ",
-        "representativeEmailAddress": "Email address - welsh ",
-        "appealDocuments": "List of forms uploaded (Appeal form) - welsh ",
-        "supportingDocuments": "List of supporting documents uploaded - welsh ",
-        "additionalInformation": "Additional information - welsh ",
-        "documentRelevance": "Document relevance - welsh "
+        "fullName": "Enw llawn ",
+        "dateOfBirth": "Dyddiad geni ",
+        "emailAddress": "Cyfeiriad e-bost ",
```
Comment (rajaram-nagarajan) [inline src/main/resources/check-your-answers/translation.json:56]: Unwanted spaces at the last can be removed.

File: src/main/resources/subject-contact-details/translation.json [hunk 2]
```diff
@@ -41,15 +41,15 @@
         },
         "cy": {
             "subjectEmailAddress": {
-                "required": "Please enter a valid email address",
-                "invalid": "Every email address consists of 3 elements: local-part, @ symbol (pronounced as “at”), and a domain name (Welsh)"
+                "required": "Nodwch gyfeiriad e-bost dilys",
+                "invalid": "Mae pob cyfeiriad e-bost yn cynnwys tair elfen: rhan lleol, symbol @ (a ynganir fel “at”), ac enw parth"
```
Comment (rajaram-nagarajan) [inline src/main/resources/subject-contact-details/translation.json:45]: interesting. don't we need to translate the words symbol and at?

File: src/main/resources/upload-supporting-documents/translation.json [hunk 3]
```diff
@@ -1,38 +1,42 @@
...
-    "uploadGuidance": "Once you have selected the file click \"Upload file\" to upload it. (Welsh)",
-    "uploadButton": "Upload file (Welsh)"
+      "serviceName": "Apelio i’r Tribiwnlys Haen Gyntaf",
+      "title": "Llwytho dogfennau ategol",
+      "line1": "Mae’n rhaid i chi anfon copi o lythyr penderfyniad adolygiad yr Awdurdod gyda ffurflen y tribiwnlys wedi’i llenwi.",
+      "uploadRequirementHeader": "Gofynion uwchlwytho ffeiliau",
+      "uploadRequirement": "<div class='govuk-body-s'><ul class=‘govuk-list govuk-list--bullet’><li>Fformatau ffeiliau: MS Word, MS Excel, PDF, JPG, GIF, PNG, TXT, RTF</li><li>Maint ffeil pob dogfen: hyd at 20 megabeit (MB)</li><li>Ni ellir llwytho ffeiliau sydd wedi’u diogelu â chyfrinair</li></ul><p> Ni allwch lwytho ffeiliau gweithredu (.exe), ffeiliau zip neu ffeiliau eraill oherwydd y risg o firws.</div>",
```
Comment (rajaram-nagarajan) [inline src/main/resources/upload-supporting-documents/translation.json:18]: Are we missing dot(.) before zip here?

File: src/main/resources/subject-contact-details/translation.json (hunk 2 above)
Comment (tomxelliott) [reply src/main/resources/subject-contact-details/translation.json:45]: symbol in Welsh is the same as in English funnily enough

File: src/main/resources/check-your-answers/translation.json (hunk 1 above)
Comment (tomxelliott) [reply src/main/resources/check-your-answers/translation.json:56]: Good spot, will resolve

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Review by JakeCG — APPROVED** [review]

**Pull request merged**